### PR TITLE
added gamespecchanges

### PR DESCRIPTION
**Pull request by PriyankaVaidya**
To check the async calls

### CODE REVIEW

File: statistics/read-rtdb.js [hunk 1]
```diff
@@ -1,15 +1,88 @@
...
+const serviceAccount = require("../../testproject-firebase-adminsdk.json");
 const admin = require("firebase-admin");
 
 admin.initializeApp({
   credential: admin.credential.cert(serviceAccount),
-  databaseURL: "https://universalgamemaker.firebaseio.com",
-  storageBucket: 'universalgamemaker.appspot.com'
+  // databaseURL: "https://universalgamemaker.firebaseio.com",
+  // storageBucket: 'universalgamemaker.appspot.com'
+  databaseURL: "https://testproject-a6dce.firebaseio.com/",
```
Comment (apsdehal) [inline statistics/read-rtdb.js:9]: Can you please make a new file instead of making so many changes in this one.

File: statistics/read-rtdb.js [hunk 2]
```diff
@@ -1,15 +1,88 @@
...
+                {
+                 promises.push(dbTargetImg.child(images.key).set(images.val()));                  
+                }
+              }); 
+            }
+          });                    
+        });  
+      }    
+    });
+    Promise.all(promises).then( "Done Adding elements" );
```
Comment (apsdehal) [inline statistics/read-rtdb.js:78]: Have you tried doing `admin.app().delete()` inside this block?

Comment (apsdehal) [reply statistics/read-rtdb.js:78]: It should work, looks fine to me.

File: statistics/read-rtdb.js (hunk 1 above)
Comment (apsdehal) [reply statistics/read-rtdb.js:9]: Copy everything to a new file.

**Comment by PriyankaVaidya** [comment]
This branch had conflicts. Closing it as a new branch has been opened to resolve the same issue

**Comment by apsdehal** [comment]
Normally, you dont need to close PR. You can force push on top of your branch to update the PR.

**Pull request closed without merging**